modules/Bio/EnsEMBL/BioMart/Mart.pm

```diff
@@ -89,7 +89,7 @@ sub genome_to_exclude {
 	else{
 		$filename = $FindBin::Bin.'/exclude_'.$division.'.ini';
 	}
-	@excluded_species = read_file( $filename,chomp => 1) if -e $filename;
+	@excluded_species = read_file( $filename,chomp => 1);
```

Comment:
Was it not the content of another PR I validated ? (to add the test against filename existence ?)

Reply:
Yes but I realised while testing this code on the Variation mart that it's dangerous to do that. If something is wrong with the path or your repo then you silently skip this part and end up building these species. I made this subroutine generic to work on all the divisions but for now I only call it for vertebrates.

Reply:
Now I am pretty sure my comment is sensible: https://github.com/Ensembl/ensembl-biomart/pull/86/files#diff-210035484e0b549a745a5df9a9dde94aR92

Reply:
That's what I changed in the check script so that this error don't come back for other divisions: https://github.com/Ensembl/ensembl-biomart/pull/88/files#diff-f1692b9444621189a7367b9a36b1264eR431